FSharpLu.Json/Compact.fs

```diff
@@ -10,18 +13,33 @@ open Microsoft.FSharp.Reflection
 type CompactUnionJsonConverter() =
     inherit Newtonsoft.Json.JsonConverter()
 
+    let processWithContractResolver (serializer:JsonSerializer) (name:string) =
```

Comment:
- I agree regarding the use of `toPascal` for `DefaultContractResolver`
- Having two Converters instead might be OK but I'd rather avoid all the OOP overhead if possible. (The general approach with FSharpLu is to stick to FP as much as possible, and use let-bindings and statically resolved typed parameters.) What's the drawback with the inlining solution?